### PR TITLE
RIA-4351 Upper Tribunal bundle config

### PR DESCRIPTION
**Pull request by bradsorour**
### JIRA link (if applicable) ###

[RIA-4351](https://tools.hmcts.net/jira/browse/RIA-4351)

### Change description ###

Bundle configuration for new IAC Upper Tribunal bundle.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/bundleconfiguration/iac-upper-tribunal-bundle-config.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        property: /hearingDocuments
+        filters:
+          - property: /tag
+            value: hearingRequirements
+      - type: documentSet
+        property: /hearingDocuments
+        filters:
+          - property: /tag
+            value: hearingNotice
+  - name: Legal representative documents
```
Comment (makowalski) [inline src/main/resources/bundleconfiguration/iac-upper-tribunal-bundle-config.yaml:44]: that will become "Appellant documents" soon, worth to confirm with Frank what we want to see in bundle

Comment (makowalski) [reply src/main/resources/bundleconfiguration/iac-upper-tribunal-bundle-config.yaml:44]: https://tools.hmcts.net/jira/browse/RIA-4510

Comment (bradsorour) [reply src/main/resources/bundleconfiguration/iac-upper-tribunal-bundle-config.yaml:44]: Correct. Frank has confirmed this collection name change to "Appellant documents" (as per his comment in AIP ticket RIA-4510). He's added similar comment to ticket RIA-4351.

**Review by makowalski — APPROVED** [review]
LGTM

**Review by yogesh-hullatti — APPROVED** [review]

**Pull request merged**